omaha/protocol/protocol.go

```diff
@@ -72,3 +72,31 @@ func DetectProtocolVersion(data []byte, contentType string) (string, error) {
 func IsJSONRequest(contentType string) bool {
 	return contentType == "application/json"
 }
+
+// IsPingbackRequest checks if the request body is a pingback.
+// For now, it only checks JSON requests for an "events" field.
+func IsPingbackRequest(body []byte, contentType string) bool {
```

Comment:
Not sure if that's needed for ping-back requests (after all, they're fire-and-forget ones), but, for the sake of consistency, we may consider doing that only for known extensions:

https://github.com/brave/go-update/blob/6e2eb1983dd0f6af468d30cda89f2e1cf5df3976/controller/controller.go#L312-L332

Currently, ping-back request for `"appid": "hfnkpimlhhgieaddgfemjhofmfblmnib"` generates a 307 redirect.